.github/workflows/docker_publish.yml

```diff
@@ -5,35 +5,21 @@ on:
   workflow_dispatch:
   push:
     paths:
-      - 'CI/Dockerfile'
-      - '.github/workflows/docker_publish.yml'
+      - "CI/Dockerfile"
+      - ".github/workflows/docker_publish.yml"
```

Comment:
out of curiosity, is this necessary for some platform?  I think both are correct.

Reply:
Ops.. It happened using the auto format